contact.html

```diff
@@ -13,7 +13,9 @@ <h2 class="logo">FELIX<br>LETHEREN</h2>
     <div class="nav-container">
         <div class="nav-bar">
             <a href="index.html" class="nav-items">Portfolio</a>
-            <a href="about.html" class="nav-items">About Me</a>
+            <a href="about.html" class="nav-items">About Me
+                <div class="pointer-arrow" id="contact-pointer"></div>
```

Comment:
Arrow links to about me section regardless of section that you're on

Reply:
Resolved!